docs/configuring-a-resource.md

```diff
@@ -11,6 +11,8 @@ documentation of the resource:
 - [Cross Resource Referencing]
 - [Additional Sensitive Fields and Custom Connection Details]
 - [Late Initialization Behavior]
+- [Overriding Terraform Resource Schema]
```

Comment:
This section has not a reference. So I added with this PR. Please also check change in the line 621